lib/ProductOpener/Redis.pm

```diff
@@ -0,0 +1,43 @@
+use Redis::Client;
+
+BEGIN
+{
+	use vars       qw(@ISA @EXPORT_OK %EXPORT_TAGS);
+	@EXPORT_OK = qw(
+		&push_to_search_service
+
+		);    # symbols to export on request
+	%EXPORT_TAGS = (all => [@EXPORT_OK]);
+}
```

Comment:
I think you should first declare your package and declare you use Export and ProductOpener::PerlStandards

@stephanegigandet knows better than me on this kind of matter :-D

```suggestion
use ProductOpener::PerlStandards;
use Exporter qw< import >;

BEGIN
{
	use vars       qw(@ISA @EXPORT_OK %EXPORT_TAGS);
	@EXPORT_OK = qw(
		&push_to_search_service
		);    # symbols to export on request
	%EXPORT_TAGS = (all => [@EXPORT_OK]);
}

use Redis::Client;
```